src/style.css

```diff
@@ -1 +1,74 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/AvenirLTStd-Book.woff2) format('woff2');
+  font-weight: 400;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+}
+
+input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  color: #3D4E61;
+}
+
+
+
+.input {
+  background-image: url(images/Icon.png);
+  background-repeat: no-repeat;
+  background-position: 5px;
+  padding-left: 30px;
+  border-radius: 3px;
+  border: 1px solid #E1E7ED;
+  font-size: 16px;
+}
+
+.input_item-1 {
+  width: 900px;
+  height: 70px;
+  top: 71px;
+  left: 48px;
```

Comment:
The 'top' and 'left' properties are used here, but they have no effect unless the element is positioned (e.g., relative, absolute, fixed). Consider removing these properties or setting a position for the element.